test/parallel/base_test_mxnet.py

```diff
@@ -176,6 +176,86 @@ def test_horovod_allreduce_inplace(self):
             assert almost_equal(tensor.asnumpy(), multiplied.asnumpy(), atol=threshold), \
                 f'hvd.allreduce produces incorrect results for self: {hvd.rank()} {count} {dtype} {dim}'
 
+    def test_horovod_allreduce_min(self):
+        """Test that the allreduce correctly finds minimum value of 1D, 2D, 3D tensors."""
+        hvd.init()
+        size = hvd.size()
+        rank = hvd.rank()
+        dtypes = self.filter_supported_types(['int32',   'int64',
+                                              'float32', 'float64'])
```

Comment:
Currently, most MXNet tests don't cover `float16`.

I guess it would be nice to extend that soon.

Reply:
Similar comment to that about PyTorch and FP16 on CPU.